test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -738,6 +738,7 @@ public void EntryPointTextToKeyToText()
                       'Name': 'Data.TextLoader',
                       'Inputs': {{
                         'InputFile': '$file'
+                        {8}
```

Comment:
I know this already existed before you change, but what do you think about using string interpolation here?  I think it would make this so much more readable:

```C#
'Inputs': {{
                        'InputFile': '$file'
                        {string.IsNullOrWhiteSpace(loader) ? "" : string.Format(",'CustomSchema': '{0}'", loader)}
```